### PR TITLE
[TRANSLATIONS] Style fixes for Estonian

### PR DESCRIPTION
**Pull request by cernodile**
## Purpose

Various style and consistency changes for Estonian localization. Also translated a handful of recently added strings.

EDIT: I'm aware of the new conflicts. I will lay low for a while until the usetup changes are finalized.

### CODE REVIEW

**Comment by cernodile** [comment]
Hopefully addressed the merge conflict.

**Comment by HBelusca** [comment]
Hi, can you please re-update your PR? (other conflicts) Thanks!

**Comment by cernodile** [comment]
@HBelusca done.

File: base/setup/reactos/lang/et-EE.rc [hunk 1]
```diff
@@ -153,23 +153,23 @@ BEGIN
     IDS_RESTARTSUBTITLE "Paigaldamise esimene järk on lõpetatud, teise järgu alustamiseks taaskäivita arvuti"
     IDS_SUMMARYTITLE "Paigaldamise kokkuvõte"
     IDS_SUMMARYSUBTITLE "Seadete kontrollimine enne paigaldamise alustamist"
-    IDS_ABORTSETUP "ReactOS ei ole veel täielikult paigaldatud. Kas tõesti lahkuda?"
+    IDS_ABORTSETUP "ReactOS ei ole veel lõplikult paigaldatud. Kas soovite tõesti lahkuda?"
     IDS_ABORTSETUP2 "Katkestada paigaldamine?"
-    IDS_NO_TXTSETUP_SIF "Unable to find 'txtsetup.sif'.\nSetup is unable to continue."
-    IDS_CAPTION "ReactOSi paigaldamine"
+    IDS_NO_TXTSETUP_SIF "Ei leitud faili 'txtsetup.sif'\nPaigaldust ei saa lõpuni viia."
```
Comment (HBelusca) [inline base/setup/reactos/lang/et-EE.rc]: The first sentence (that ends with 'txtsetup.sif') should end with a full stop.

File: base/setup/reactos/lang/et-EE.rc [hunk 2]
```diff
@@ -2,23 +2,23 @@ LANGUAGE LANG_ESTONIAN, SUBLANG_DEFAULT
 
 IDD_STARTPAGE DIALOGEX 0, 0, 317, 193
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "ReactOS Setup"
+CAPTION "ReactOS'i paigaldamine"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Tere tulemast ReactOSi paigaldamisprogrammi", IDC_STARTTITLE, 115, 8, 195, 24
-    LTEXT "Selle CD pealt ei saa praegu veel ReactOSi paigaldada! ReactOSi paigaldamiseks on tarvis arvuti taaskäivitada selle CD-ga.", IDC_STATIC, 115, 40, 195, 100
+    LTEXT "Tere tulemast ReactOS'i paigaldamus viisardisse.", IDC_STARTTITLE, 115, 8, 195, 24
```
Comment (HBelusca) [inline base/setup/reactos/lang/et-EE.rc]: No ending stop (that's the banner)

**Pull request merged**